README.md

```diff
@@ -1,43 +1,56 @@
-
-Our platform is a unified solution for continuous delivery and full-stack experimentation. Split unifies DevOps and product management, helping agile engineering and product teams accelerate the pace of product delivery and make data-driven decisions, through our robust feature flagging and extensive experimentation capabilities. With Split, organizations can now accelerate time to value, mitigate risk, and drive better outcomes, all in a unified platform.
-
-To learn more about Split, contact hello@split.io, or start a 14-day trial at https://www.split.io/signup/.
-
-Split has built and maintains a SDKs for:
-
-* Java [Github](https://github.com/splitio/java-client) [Docs](http://docs.split.io/docs/java-sdk-guide)
-* Javascript [Github](https://github.com/splitio/javascript-client) [Docs](http://docs.split.io/docs/javascript-sdk-overview)
-* Node [Github](https://github.com/splitio/javascript-client) [Docs](http://docs.split.io/docs/nodejs-sdk-overview)
-* .NET [Github](https://github.com/splitio/.net-client) [Docs](http://docs.split.io/docs/net-sdk-overview)
-* Ruby [Github](https://github.com/splitio/ruby-client) [Docs](http://docs.split.io/docs/ruby-sdk-overview)
-* PHP [Github](https://github.com/splitio/php-client) [Docs](http://docs.split.io/docs/php-sdk-overview)
-* Python [Github](https://github.com/splitio/python-client) [Docs](http://docs.split.io/docs/python-sdk-overview)
-* GO [Github](https://github.com/splitio/go-client) [Docs](http://docs.split.io/docs/go-sdk-overview)
-* Android [Github](https://github.com/splitio/android-client) [Docs](https://docs.split.io/docs/android-sdk-overview)
-* IOS [Github](https://github.com/splitio/ios-client) [Docs](https://docs.split.io/docs/ios-sdk-overview)
-
-For a comprehensive list of opensource projects visit our [Github page](https://github.com/splitio?utf8=%E2%9C%93&query=%20only%3Apublic%20).
-
+# Split SDK for iOS
+[![Build Status](https://travis-ci.com/splitio/javascript-client.svg?branch=master)](https://travis-ci.com/splitio/ios-client)
+
+## Overview
+This SDK is designed to work with Split, the platform for controlled rollouts, which serves features to your users via a Split feature flag to manage your complete customer experience.
+ 
+[![Twitter Follow](https://img.shields.io/twitter/follow/splitsoftware.svg?style=social&label=Follow&maxAge=1529000)](https://twitter.com/intent/follow?screen_name=splitsoftware)
+ 
+## Compatibility
+This SDK is compatible with iOS deployment target versions 9.0 and later, and Swift version 4 and later.
+
+## Getting started
+Below is a simple example that describes the instantiation and most basic usage of our SDK:
+```
+<Factory instantiation>
+<Block until Ready>
+<Get treatment>
+<If treatment == ...>
+```
```

Comment:
@javrudsky please complete this section and remove placeholder tags. This example would be as simple as possible, minimal code to get a treatment and use it on the ifs. 
See for guidance:
https://help.split.io/hc/en-us/articles/360020401491-iOS-SDK#2-instantiate-the-sdk-and-create-a-new-split-client
https://help.split.io/hc/en-us/articles/360020401491-iOS-SDK#basic-use